plugins/irma-web/dom-manipulations.js

```diff
@@ -109,7 +109,9 @@ module.exports = class DOMManipulations {
   _stateShowingIrmaButton() {
     return `
       <!-- State: ShowingButton -->
-      <button class="irma-web-button" data-irma-glue-transition="openIrmaApp">${this._translations.button}</button>
+      <a id="irma-web-button-link" data-irma-glue-transition="openIrmaApp">
```

Comment:
You changed the transition to `codeScanned`, above. I'm not sure why you did that, but if you do, this needs to change too :)

Reply:
(or maybe it can just go)

Reply:
Ah yeah right. I did this, because there is no real difference anymore between the both transition. Because an <a> link has to be used (because iOS will otherwise not allow opening the link), opening the app is pretty much the same as scanning a QR. Therefore distinguishing between both operations is a bit superfluous. Then we have to say in the `irma-server` 'if it was a button you have to send 'openIrmaApp' and if it was a QR you have to send 'codeScanned'. That's not necessary at all.
Of course you could argue that another name of this transition would be better now, for example `appConnected` or something. That would better describe the transition.

Reply:
But indeed it can be fully deleted.